web/template/donor/make_a_new_lpa.gohtml

```diff
@@ -8,6 +8,7 @@
             <h1 class="govuk-heading-xl">{{ tr .App "makeANewLPA" }}</h1>
 
             {{ trHtml .App "youShouldOnlyChangeYourDetailsIfTheyNeedToBeUpdated" }}
+            {{ trHtml .App "makingAnLPAForSomeoneElseDetails" }}
```

Comment:
I didn't know we'd got a details component in a string, could it be extracted to use the template instead? Or should I not care?

Reply:
No, we should be consistent, I'll extract to use the component.